commit.go

```diff
@@ -42,17 +40,17 @@ func (cm *Commit) IsEmpty() bool {
 }
 
 // Path returns the internal path of this commitMsg
-func (cm *Commit) Path() datastore.Key {
+func (cm *Commit) Path() string {
 	return cm.path
 }
 
 // SetPath sets the internal path property of a commit
 // Use with caution. most callers should never need to call SetPath
 func (cm *Commit) SetPath(path string) {
 	if path == "" {
```

Comment:
This function can be collapsed to:

```
cm.path = path
```

Reply:
Lovely. I'm going to skip this b/c path getter & setters are going to die in a subsequent PR